checkov/cdk/checks/python/APIGatewayCacheEnable.yaml

```diff
@@ -0,0 +1,18 @@
+metadata:
```

Comment:
Check, performencewise - if it's better to do it as following: 
```
and: 
  - pattern: aws_cdk.aws_apigateway.Stage(<ANY>)
  - not_pattern: aws_cdk.aws_apigateway.Stage(<ANY>, cache_cluster_enabled=True, <ANY>)
```

Reply:
@achiar99

Reply:
@RabeaZr Notice - This comment is relevant to all the other patterns as well, so it's important.